### DISCUSSION TITLE
Make "featured talks" embeddable via iFrame

### DISCUSSION THREAD

**Discussion by saschafoerster** [pretalx/pretalx#1200] [Ideas]
I would like to embed the "featured talks" page into another website, so people get a preview directly on the event-website. 
Right now, I get this error: 
`Refused to display 'https://pretalx.com/event/featured/' in a frame because it set 'X-Frame-Options' to 'DENY'.`
Some responsive embed code in the backend would be really helpful (like the widgets for Pretix-ticket-shops).

**Comment by rixx** [comment]
pretalx currently provides widgets, but only for the two different schedule views (grid, or list). Adding a new widget for featured talks would be quite a bit of effort, though we would of course welcome a pull request doing so.

Embedding the HTML directly likely would not produce the intended effect, since the surrounding content (header, log-in menu, and so on) would be look quite odd on an external website.
